docs/network/governance.md

```diff
@@ -65,7 +65,13 @@ entities such as Accelerators who utilize the Veres One Blockchain to provide
 services to customers. Community Group decisions are made by consensus and sent
 to the Board of Governors for review and voting.
 
-## Board of Governors Checks and Balances
+Since the Veres One Community Group is consensus based Community Group
+participants are checked by the other participants in the Community Group.
+There is also a final check on Community Group decisions by the Board of
+Governors who accept or reject Community Group decisions based on whether they
+comply with the Veres One Mission. 
```

Comment:
I think we should strike this whole paragraph. We've already set decisions are made by consensus above and that the output of the CG goes to the Board of Governors [for a final check]. The only new content is a note about how decisions are made -- which is whether or not the Board of Governors thinks that the consensus decision complies with the Veres One Mission. If that's necessary to say we should just add another sentence to the top paragraph:

"The Board of Governors will accept or reject Community Group decisions based on their compliance with the Veres One Mission."

Reply:
Yes, agreed, favor @dlongley's wording.